SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -56,6 +57,18 @@ struct ConfigureConnection
 		Params.network.modular_kcp.upstream_heartbeat = &HeartbeatParams;
 #endif
 
+		// There is no need to encrypt UnrealWorker (server) to runtime traffic as these machines will always be in the same data center.
+		if (*Config.WorkerType != SpatialConstants::DefaultServerWorkerType.ToString() && GetDefault<USpatialGDKSettings>()->bUseSecureConnection)
```

Comment:
Are we confident that's not going to change? I can approve using `bUseSecureConnection` flag, however switching the encryption depending on the WorkerType seems a bit unsafe. What if someone changes the WorkerType or the location of the machines?

Reply:
We're quite sure this won't change. We will document this regardless.